### DISCUSSION TITLE
eForm40 validation error

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#412] [Q&A] [answered]
We get the following error when validating eForm40:
If Procedure Legal Basis (BT-01) value is equal to 'Directive 2014/23/EU', then Notice Type (BT-02) value must be equal to ('Prior information notice or a periodic indicative notice used as a call for competition – social regime' or 'Contract or concession notice – standard regime' or 'Voluntary ex-ante transparency notice' or 'Contract or concession award notice – standard regime' or 'Contract or concession award notice – social regime').

If I understand correctly the validation rule says that if the BT-01 value is equal to 'Directive 2014/23/EU' then the notice type should be the following:
14 (pin-cfc-social)
19 (cn-standard)
28 (veat)
32 (can-standard)
35 (can-social)

This error doesn't seem right in this case because if we look at this file: https://github.com/OP-TED/eForms-SDK/blob/main/notice-types/notice-types.json, we can see that notice type 40 also has 'Directive 2014/23/EU' as its legal basis
```
{
    "documentType" : "CAN",
    "legalBasis" : "32014L0023",
    "formType" : "cont-modif",
    "type" : "can-modif",
    "description" : "Contract modification notice – concessions directive",
    "subTypeId" : "40",
    "_label" : "notice|name|40",
    "viewTemplateIds" : [ "40", "summary" ]
  }
```
Can you explain why we get this error for eForm40 if everything should be correct according to the documentation?
We should be able to do contract modifications to concession contracts.

**Comment by EEanalyst** [comment]
We are on version 1.3

**Comment by YvesJo** [comment, marked as answer]
There is indeed a mistake in the initial rule statement and this will get fixed in a later version.

**Comment by bertrand-lorentz** [comment]
For your information, this correction is included in SDK 1.3.5, which has just been released.
